controllers/ebpf/agent_controller.go

```diff
@@ -450,7 +450,7 @@ func (c *AgentController) setEnvConfig(coll *flowslatest.FlowCollector) []corev1
 	}
 
 	sampling := coll.Spec.Agent.EBPF.Sampling
-	if sampling != nil && *sampling > 1 {
+	if sampling != nil && *sampling > 0 {
```

Comment:
@msherif1234 was that on purepose ?

It seems legit to me to pass sampling even when `1`

Reply:
that was b4 my time if u check the doc on the agent it says both 0 and 1 treated the same I pointed @Amoghrd to this when he told me about the sampling metrics

Reply:
Is it good to merge this PR then?

Reply:
not very important but if we allow sampling=1 in agent then there's a little optimization to do: https://github.com/netobserv/netobserv-ebpf-agent/pull/313